jupyter/minimal/ubi8-python-3.8/requirements.in

```diff
@@ -1 +1,60 @@
+# Minimal notebook requirements
 jupyterlab==3.2.4
+jupyter_server==1.4.1
+notebook==6.4.12
+setuptools==65.4.0
+pip==22.2.2
+
+# JupyterLab extensions
+# A JupyterLab extension for version control using git
+jupyterlab-git==0.39.2
+# Jupyter Kernel Gateway is a web server that supports different mechanisms for spawning and communicating with Jupyter kernels.
+jupyter_kernel_gateway==2.4.0
+# JupyterLab extension for dependency management and optimization created to guarantee reproducibility of Jupyter notebooks.
+jupyterlab-requirements>=0.6.4
+# This extension provides control over the notebook dependencies.The main goals of the project are the following: 1.manage notebook requirements without leaving the notebook 2.provide a unique and optimized* environment for each notebook
+jupyter-nbrequirements==0.7.3
+# The jupyter_nbextensions_configurator jupyter server extension provides graphical user interfaces for configuring which nbextensions are enabled (load automatically for every notebook), and display their readme files.
+jupyter_nbextensions_configurator==0.5.0
+# Contains a collection of extensions that add functionality to the Jupyter notebook
+jupyter-contrib-nbextensions==0.5.1
+
+# Additional requirements for CPAAS build
+# Tools to help build and install Jupyter Python packages that require a pre-build step that may include JavaScript build steps.
+jupyter_packaging~=0.9
+# JupyterLab Server sits between JupyterLab and Jupyter Server, and provides a set of REST API handlers and utilities that are used by JupyterLab.
+jupyterlab_server==2.3.0
+# The Jupyter NbClassic is a web application that allows you to create and share documents that contain live code, equations, visualizations, and explanatory text
+nbclassic==0.2.6
+# IPython provides a rich toolkit to help you make the most out of using Python interactively.
+ipython==7.16.1
+ipykernel==5.5.5
+# Cryptography is a package which provides cryptographic recipes and primitives to Python developers
+cryptography==3.3.1
+# Dictdiffer is a library that helps you to diff and patch dictionaries.
+dictdiffer==0.5.0
+# Flit is a simple way to put Python packages and modules on PyPI.
+flit==3.4.0
+flit-core==3.4.0
```

Comment:
These are CPaaS dependencies and we are not using CPaaS so we don't really need them.